src/AnalyzerConfiguration/NI.ruleset

```diff
@@ -174,20 +171,8 @@
     <Rule Id="RS1012" Action="None" />
     <Rule Id="RS1013" Action="None" />
     <Rule Id="RS1014" Action="None" />
-  </Rules>
-  <Rules AnalyzerId="Microsoft.QualityGuidelines.Analyzers" RuleNamespace="Microsoft.QualityGuidelines.Analyzers">
-    <Rule Id="CA1802" Action="None" />
-    <Rule Id="CA1814" Action="None" />
-    <Rule Id="CA1821" Action="Warning" />
-    <Rule Id="CA1822" Action="None" />
-    <Rule Id="CA2119" Action="Warning" />
-    <Rule Id="CA2214" Action="Warning" />
-    <Rule Id="CA2219" Action="Warning" />
-  </Rules>
-  <Rules AnalyzerId="Microsoft.QualityGuidelines.CSharp.Analyzers" RuleNamespace="Microsoft.QualityGuidelines.CSharp.Analyzers">
-    <Rule Id="CA2000" Action="None" />
-    <Rule Id="CA2109" Action="None" />
-    <Rule Id="CA2200" Action="Warning" />
+    <Rule Id="RS0015" Action="None" />
```

Comment:
Re-saving the ruleset rearranges these to be first since their number is lower than RS1001. It also corrects RS1004/1009. So, it would be nice if these were ordered correctly.

Reply:
I misread them and thought that 0015 was 1015. I have ordered everything in this section correctly